plugins/msg-broker/mcollective/lib/openshift/mcollective_application_container_proxy.rb

```diff
@@ -2413,11 +2427,13 @@ def rsync_destination_container(gear, destination_container, destination_distric
       log_debug "DEBUG: Gear platform is '#{platform}'"
       destination_avail_space = destination_container.get_node_disk_free
       destination_total_space = destination_container.get_node_total_size
+      destination_df_buffer = destination_container.get_node_df_buffer.to_f
```

Comment:
The `.to_f` here is kind of superfluous since `get_node_df_buffer` already does it.